lib/gtl_formatter.rb

```diff
@@ -177,9 +178,22 @@ def self.os_info_details(item)
     "svg/os-#{item.os_image_name}.svg"
   end
 
+  def self.item_provider(item)
```

Comment:
yea, this is too deep.

Need to modify the constants that are used for mapping. Would be best if they call this method with the correct type